cmd/skopeo/utils.go

```diff
@@ -147,15 +148,23 @@ func (opts *imageOptions) newSystemContext() (*types.SystemContext, error) {
 	if opts.noCreds {
 		ctx.DockerAuthConfig = &types.DockerAuthConfig{}
 	}
+	cf, err := compression.AlgorithmByName("gzip")
```

Comment:
This shouldn’t hard-code a Skopeo-specific default; the default in c/image should be used.

Reply:
We have a chicken-egg problem now as we agreed to add the constants with https://github.com/containers/image/pull/563.

Reply:
- Skopeo shouldn’t hard-code _any_ default; so it should not need the constants (except, maybe, I suppose, to list the right names of algorithms in a help message, if one wanted to be extremely pedantic about not hard-coding that).
- Even if this did need to use the constants, how is that a chicken-egg problem? It seems to be just a simple one-directional dependency; add constants, add options to skopeo. Or is the CI now broken because c/image has merged a change without having a skopeo PR actually ready, or something? (I‘m afraid I’ve been in meetings for the past two days and I’m a bit out of the loop.)